MapboxNavigation/NavigationMapView.swift

```diff
@@ -49,7 +49,7 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     /**
      The object that acts as the navigation delegate of the map view.
      */
-    public weak var navigationMapDelegate: NavigationMapViewDelegate?
+    @objc public weak var navigationMapDelegate: NavigationMapViewDelegate?
```

Comment:
Should we rename this property to `navigationMapViewDelegate` while we’re at it? The lack of “view” in the property name has always bugged me. If not, we can take care of it as tail work.